templates/tfengine/components/cicd/variables.tf

```diff
@@ -78,7 +112,34 @@ variable "envs" {
       })
     })
   }))
-  description = {{schemaDescription $props.envs.description}}
+  description = <<EOF
+    {{schemaDescription $props.envs.description}}
+  
+    Fields:
+
+    * branch_name = {{$envsProps.branch_name.description}} 
+    * managed_dirs = {{$envsProps.managed_dirs.description}} 
+    * name = {{$envsProps.name.description}} 
+    * triggers = {{$envsProps.triggers.description}} 
+    ** apply = {{$triggerProps.apply.description}}
+    *** skip = Whether or not to skip creating trigger resources.
+    *** run_on_push = {{$triggerProps.apply.properties.run_on_push.description}}
+    *** run_on_schedule = {{$triggerProps.apply.properties.run_on_schedule.description}}
+    ** plan = {{$triggerProps.plan.description}}
+    *** skip = Whether or not to skip creating trigger resources.
+    *** run_on_push = {{$triggerProps.plan.properties.run_on_push.description}}
+    *** run_on_schedule = {{$triggerProps.plan.properties.run_on_schedule.description}}
+    ** validate = {{$triggerProps.validate.description}}
+    *** skip = Whether or not to skip creating trigger resources.
+    *** run_on_push = {{$triggerProps.validate.properties.run_on_push.description}}
+    *** run_on_schedule = {{$triggerProps.validate.properties.run_on_schedule.description}}
```

Comment:
I think spaces (2 per level) would look better than multiple asterisks

Reply:
You're right too, also that's what we agreed for other recipes.

Fixed